README.md

```diff
@@ -69,15 +60,14 @@ The goal of this project is to make the easiest, fastest, and most painless way
 
 ## Installation
 
-Make sure you install the [prerequisites](https://gogs.io/docs/installation) first.
+**Note: As Gitea is a [Gogs](https://github.com/gogits/gogs) fork, tutorials and documentation related to gogs applies to Gitea too**
 
-There are 5 ways to install Gogs:
+There are 2 ways to install Gitea:
```

Comment:
Drop the count, those ways are going to change and we'll forget to update the count :)